src/documents/showcase/Willy and Mathilda's Houseboat Adventure꞉ The Game.md

```diff
@@ -0,0 +1,7 @@
+```
+layout: showcase
+title: "Willy and Mathilda's Houseboat Adventure? The Game"
```

Comment:
That is what was intended, and actually I had copied it from somewhere else.  I guess it didn't survive the copy-paste?  Or is it just not displaying properly?